docker/nginx.conf

```
@@ -70,7 +70,7 @@ http {
     limit_req_zone $rate_limit_key zone=web_limit:10m rate=1r/s;
     # Higher rate for APIs since they are cheaper and we often hit them
     # multiple times per page load.
-    limit_req_zone $rate_limit_key zone=api_limit:10m rate=200r/m;
+    limit_req_zone $rate_limit_key zone=api_limit:10m rate=3r/s;
```

Rolled this to be per minute, it's blocking librarian requests #10875 
```suggestion
    limit_req_zone $rate_limit_key zone=api_limit:10m rate=180r/m;
```